docs/DeveloperGuide.md

```diff
@@ -90,29 +90,30 @@ Here's a (partial) class diagram of the `Logic` component:
 
```

Comment:
![image](https://github.com/AY2324S1-CS2103T-W15-2/tp/assets/94535035/c70689fd-96f2-490b-be68-8c1a41931ccf)
Change the link to our repo one

Reply:
Good catch Ming Yuan. The link is now the updated version.